platform/android/export/export_plugin.cpp

```diff
@@ -787,6 +787,12 @@ void EditorExportPlatformAndroid::_get_permissions(const Ref<EditorExportPreset>
 				r_permissions.push_back("com.oculus.permission.HAND_TRACKING");
 			}
 		}
+		bool use_anchor_api = p_preset->get("xr_features/use_anchor_api");
```

Comment:
Isn't this something we can simply have in the plugin? Assuming it is build as an AAR it's able to add to the manifest if I'm not mistaken, and the inclusion of the plugin would require this to be true.

Reply:
@konczg Take a look at https://github.com/godotengine/godot/pull/78958; in 4.2, we are expanding the capabilites of the android plugin to allow moving these type of configuration out of core.
You can see what the refactor looks like in https://github.com/GodotVR/godot_openxr_loaders/pull/38

So you should move these configs, as well as the anchor gdextension implementation in the Godot Openxr loader plugin instead of including them in core.